db/schema.rb

```diff
@@ -10,16 +10,17 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20160718220652) do
+ActiveRecord::Schema.define(version: 20160719222657) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
+  enable_extension "citext"
 
   create_table "customers", force: :cascade do |t|
-    t.string   "first_name"
-    t.string   "last_name"
     t.datetime "created_at"
     t.datetime "updated_at"
+    t.citext   "first_name"
+    t.citext   "last_name"
```

Comment:
I noticed this was the only model modified to include CITEXT. Was this the only model in your scope that needed it? I didn't see any others, but just in case wanted to bring attention to it.